mw-webapp/src/component/tooltip/Tooltip.tsx

```diff
@@ -0,0 +1,39 @@
+import {PropsWithChildren, ReactNode} from "react";
+import clsx from "clsx";
+import {PositionTooltip} from "src/component/tooltip/PositionTooltip";
+import styles from "src/component/tooltip/Tooltip.module.scss";
+
+/**
+ * Describe props component Tooltip
+ */
+interface TooltipProps {
+  /**
+  * Tooltip's content
+  */
+  content: string | ReactNode;
+  /**
+  * Tooltip's className
+  * default ("tooltip")
```

Comment:
it will not be tooltip because you are using module.scss. Lets incapsulate this data. I added github suggestion

Reply:
I get it)